CMakeLists.txt

```diff
@@ -172,7 +173,9 @@ if( BUILD_WITH_TENSILE )
   option( Tensile_PRINT_DEBUG "Tensile to print runtime debug info?" OFF )
 
   set( Tensile_TEST_LOCAL_PATH "" CACHE PATH "Use local Tensile directory instead of fetching a GitHub branch" )
-  set_property( CACHE Tensile_LOGIC PROPERTY STRINGS asm_full asm_lite hip_lite other )
+  set( CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended onto install directories" )
+  set_property( CACHE Tensile_ARCHITECTURE PROPERTY STRINGS arcturus fiji vega10 vega20 _)
```

Comment:
Should this be gfx803, gfx900, gfx906, gfx908 in place of arcturus fiji vega10 vega20 ?

Reply:
Yes it should be, thanks for catching that. I've fixed it in my latest commit